src/settings/App.tsx

```diff
@@ -8,7 +8,7 @@ import './App.css';
 import {SubmitInput} from "./SubmitInput";
 import {SortArrow} from "./SortArrow";
 import FlipMove from "react-flip-move";
-import AsyncSelect from 'react-select/async'
+import {AsyncSelect, Select} from 'react-select';
```

Comment:
react-select does not export those at the root so if you need both you will need:

```suggestion
import Select from 'react-select';
import AsyncSelect from 'react-select/async';
```

Reply:
I've just figured it out 😭 😭 😭 
Thanks anyway!